### PR TITLE
Missing capitalisation of 'must'

### PR DESCRIPTION
**Pull request by MikeRalphson**

### CODE REVIEW

**Comment by ePaul** [comment]
If you are at it, could you fix the other occurrence of `must` in the document (in the OpenAPI object's field table, tags).

**Comment by MikeRalphson** [comment]
@ePaul if you mean

> Not all tags that are used by the [Operation Object](#operationObject) must be declared

I think that "not all .. must be declared" isn't a case of MUST or MUST NOT. **But** it may be better to reword this as

> Not all tags that are used by the [Operation Object](#operationObject) need to be declared

to avoid the ambiguity.

**Comment by ePaul** [comment]
You are right, I somehow didn't read the full sentence. Your rewording seems fine.

**Comment by MikeRalphson** [comment]
Happy to fold in the other change if required.
